src/Symfony/Component/Notifier/Bridge/Sendinblue/README.md

```diff
@@ -0,0 +1,26 @@
+Sendinblue Notifier
+==============
+
+Provides Sendinblue integration for Symfony Notifier.
+
+DSN example
+-----------
+
+```
+// .env file
+SENDINBLUE_MOBILE_DSN=sendinblue://API_KEY@default?sender=PHONE
```

Comment:
Why not use SENDINBLUE_DSN instead?

Reply:
Because if an end user have also the sendinblue mailer, it can be confusing I think

Reply:
But for the mailer, the env var is always named `MAILER_DSN`, right?

Reply:
You got a point. Didn't remember that

Reply:
done